Rubrik/Private/Get-RubrikAPIData.ps1

```diff
@@ -180,6 +180,25 @@ function Get-RubrikAPIData($endpoint)
         Success     = '200'
       }
     }
+    'Get-RubrikDatabaseMount'            = @{
```

Comment:
Can you go back and arrange these entries by alphabetic order? Makes it easier to find them in the future.